vms/platformvm/state/disk_staker_diff_iterator.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+const (
+	// diffKey = [subnetID] + [inverseHeight] + [nodeID]
+	startDiffKeyLength  = ids.IDLen + database.Uint64Size
+	diffKeyLength       = startDiffKeyLength + ids.NodeIDLen
+	diffKeyHeightOffset = ids.IDLen
+	diffKeyNodeIDOffset = diffKeyHeightOffset + database.Uint64Size
+
+	// weightValue = [isNegative] + [weight]
+	weightValueLength = 1 + database.Uint64Size
```

Comment:
Should `1` be a constant in this context given everything else is?

Reply:
I refactored some of the DB constants to avoid the re-implementation here.